ui_framework/components/tabs/tabs.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { KuiTab } from './tab';
+
+export const KuiTabs = ({
+  children,
+  selectedTabIndex,
+  onSelectedTabChanged,
+  className,
+  ...rest
+}) => {
+  const classes = classNames('kuiTabs', className);
+  const tabs = children.map((child,index)=>(
+    <KuiTab
+      key={index}
+      onClick={() => onSelectedTabChanged(index)}
+      isSelected={index === selectedTabIndex}
+    >
+      {child}
+    </KuiTab>
+  ));
+
+  return (
+    <div
+      className={classes}
+      {...rest}
+    >
+      {tabs}
+    </div>
+  );
+};
+
+const selectedTabIndexCheck = (props, propName, componentName) => {
+  const childrenLength = props.children ? props.children.length : 0;
+  const selectedTabIndex = props.selectedTabIndex;
```

Comment:
The function arguments are given by React and says: let check this property in
the context of all properties. I think we can assume that the user of this lib know
this feature of React. If this a problem than probably some general documentation would
be better than using React other than recommended.
I add a check for selectedTabIndex: it must be an integer.